app/controllers/items_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ItemsController < ApplicationController
+
+  def create
+    @user = User.find(params[:user_id])
+    @item = @user.items.build(item_params)
+    @item.user = current_user
+
+    if @item.save
```

Comment:
You should also do something like `@item.update(item_params)` here instead of line 5